engine/src/main/java/org/terasology/rendering/opengl/fbms/DisplayResolutionDependentFBOs.java

```diff
@@ -118,4 +118,8 @@ public void swapReadWriteBuffers() {
         fboLookup.put(WRITEONLY_GBUFFER, fbo);
         notifySubscribers();
     }
+
+    public FBO.Dimensions getFullScale() {
+        return fullScale;
+    }
```

Comment:
This was an oversight, I earlier tried to do fbo.size / scale, and then I relized it's more efficient to do display.size. I'll fix this!